pkg/vminetworkscontroller/vmi_controller.go

```diff
@@ -158,10 +159,6 @@ func (r *VirtualMachineInstanceReconciler) vmiNetworksClaimingIPAM(
 ) (map[string]string, error) {
 	vmiNets := make(map[string]string)
 	for _, net := range vmi.Spec.Networks {
-		if net.Pod != nil {
-			continue
-		}
-
 		if net.Multus != nil && !net.Multus.Default {
```

Comment:
what about primary where it is `net.Multus.Default`
maybe we dont support it yet, would we like in the future ?

Reply:
Support that for primary UDN involve a lot of changes at different projects, so does not make sense to support it just here without sync with the others.

Reply:
should a release note be added?

Reply:
@maiqueb 
you might want to add note about it on https://github.com/openshift/enhancements/pull/1673 please ?